XamlControlsGallery/ControlPages/PipsPagerPage.xaml

```diff
@@ -0,0 +1,85 @@
+            <local:ControlExample.Example>
+                <muxc:PipsPager x:Name="TestPipsPager1"/>
+            </local:ControlExample.Example>
+            <local:ControlExample.Options>
+                <StackPanel>
+                    <ComboBox x:Name="OrientationComboBox" Header="Orientation" SelectionChanged="OrientationComboBox_SelectionChanged" SelectedValue="Horizontal">
+                        <x:String>Horizontal</x:String>
+                        <x:String>Vertical</x:String>
+                    </ComboBox>
+                    <ComboBox x:Name="PrevButtonComboBox" Header="Previous Button Visibility" SelectionChanged="PrevButtonComboBox_SelectionChanged" SelectedValue="Visible">
+                        <x:String>Visible</x:String>
+                        <x:String>VisibleOnPointerOver</x:String>
+                        <x:String>Collapsed</x:String>
+                    </ComboBox>
+                    <ComboBox x:Name="NextButtonComboBox" Header="Next Button Visibility" SelectionChanged="NextButtonComboBox_SelectionChanged" SelectedValue="Visible">
+                        <x:String>Visible</x:String>
+                        <x:String>VisibleOnPointerOver</x:String>
+                        <x:String>Collapsed</x:String>
+                    </ComboBox>
+                </StackPanel>
+            </local:ControlExample.Options>
+            <local:ControlExample.Xaml>
+                <x:String xml:space="preserve">
+&lt;muxc:PipsPager
+    Orientation="$(Orientation)"
+    PreviousButtonVisibility="$(PrevButton)"
+    NextButtonVisibility="$(NextButton)" /&gt;
+                </x:String>
+            </local:ControlExample.Xaml>
+            <local:ControlExample.Substitutions>
+                <local:ControlExampleSubstitution Key="Orientation" Value="{x:Bind OrientationComboBox.SelectedValue, Mode=OneWay}" />
+                <local:ControlExampleSubstitution Key="PrevButton" Value="{x:Bind PrevButtonComboBox.SelectedValue, Mode=OneWay}" />
+                <local:ControlExampleSubstitution Key="NextButton" Value="{x:Bind NextButtonComboBox.SelectedValue, Mode=OneWay}" />
+            </local:ControlExample.Substitutions>
+        </local:ControlExample>
+
+        <local:ControlExample x:Name="Example2" HeaderText="PipsPager integrated with a FlipView">
+            <local:ControlExample.Example>
+                <StackPanel>
+                    <FlipView x:Name="Gallery" MaxWidth="400" Height="270" ItemsSource="{x:Bind Pictures}">
+                        <FlipView.ItemTemplate>
+                            <DataTemplate>
+                                <Image Source="{Binding}"/>
+                            </DataTemplate>
+                        </FlipView.ItemTemplate>
+                    </FlipView>
+                    <muxc:PipsPager x:Name="FlipViewPipsPager"
+                              Margin="140, 10, 0, 0"
```

Comment:
Magic numbers 😱 - should we have some guidance that says "put your pager x,y,z away from the paging element"?

Reply:
Shouldn't PipsPager be horizontally centered relative to the FlipView? I know you can't do that in a vertical StackPanel, may just need to replace it with a Grid with rows.

Reply:
I was able to horizontally center the PipsPager in the vertical StackPanel. The magic "top" margin number does still exist.

Reply:
Updated to 12px to stay in the 4px grid, are there other controls that have recommendations in spacing between it and another control?

Reply:
As a rule of thumb, 12px is a good amount of space between controls on a page, 24px is a good margin around the exterior of a page (between content and edge of window). You may want to use 24px in a page when establishing visual groupings via whitespace.